packages/select/src/types.ts

```diff
@@ -66,15 +76,34 @@ export type RematchSelect<M extends object = Models, RootState = any> = ((
 	StoreSelectors<RootState>
 
 declare module '@rematch/core' {
-	interface Model<
+	export interface Model<
 		TModels extends Models<TModels> = Record<string, any>,
 		TState = any,
 		TBaseState = TState
 	> {
+		name?: string
+		state: TState
 		selectors?: ModelSelectorsConfig<TState>
+		reducers?: ModelReducers<TState>
+		baseReducer?: ReduxReducer<TBaseState>
+		effects?: ModelEffects<TModels> | ModelEffectsCreator<TModels>
+	}
+
+	export interface ModelResult<S, R, BR, E> {
```

Comment:
what is the ModelResult used for?

Reply:
for nothing, on a first instance i've tried to use it in createModel in core, and extend the type here with selectors